### PR TITLE
CONSOLE-4380: Use AlertsRulesDetailPage from `monitoring-plugin`

### PR DESCRIPTION
**Pull request by logonoff**
/hold

https://github.com/openshift/console/pull/14596
https://github.com/openshift/monitoring-plugin/pull/286

**Fixes**: 

https://issues.redhat.com/browse/CONSOLE-4380
https://issues.redhat.com/browse/OU-561

**Solution Description**: 

- Remove `AlertRulesDetailsPage` from console
- Add a route in monitoring-plugin to enable it
- Remove code related to routes that are no longer provided by console repo

**Screen shots / Gifs for design review**: 

image

The only change to the AlertsRulesDetailPage in the developer perspective is that it will no longer have a namespace selector. Since the selected alert is already namespaced, it didn't make sense to swap namespaces while focused on a specific AlertRule.

**Unit test coverage report**: 

n/a

**Test setup:**

Host the monitoring dynamic plugin locally by running
```sh
./bin/bridge -plugins monitoring-plugin=http://localhost:9001 -i18n-namespaces=plugin__monitoring-plugin -branding openshift  # console backend
yarn dev  # console/frontend and monitoring-plugin/web
```

**Browser conformance**: 

- [x] Chrome
- [x] Firefox
- [ ] Safari

### CODE REVIEW

**Comment by juzhao** [comment]
tested with cluster-bot
`launch 4.19,openshift/monitoring-plugin#286,openshift/console#14596 aws`

found 3 regression bugs, not related to these 2 PRs
https://issues.redhat.com/browse/OCPBUGS-37547
https://issues.redhat.com/browse/OCPBUGS-45892
https://issues.redhat.com/browse/OCPBUGS-45896

**Comment by logonoff** [comment]
adding labels as this is debt
/label px-approved
/label docs-approved

PR is a blocker for PF6 migration
/label acknowledge-critical-fixes-only

File: frontend/packages/dev-console/console-extensions.json [hunk 1]
```diff
@@ -1288,14 +1288,6 @@
...
     }
   },
-  {
-    "type": "console.page/route",
-    "properties": {
-      "exact": false,
-      "path": ["/dev-monitoring/ns/:ns/rules/:id"],
-      "component": { "$codeRef": "monitoring.MonitoringAlertsRulesDetailsPage" }
-    }
-  },
```
[the remark this answers is not in the document]
Comment (logonoff) [reply frontend/packages/dev-console/console-extensions.json:1298]: I just did a quick search in the codebase for `dev-monitoring` and couldn't find any corresponding routes matching this route

**Review by spadgett — APPROVED** [review]
/approved

File: frontend/public/components/monitoring/alerting.tsx [hunk 2]
```diff
@@ -1,63 +1,41 @@
...
   Modal,
   ModalVariant,
-  Popover,
-  Toolbar,
-  ToolbarContent,
-  ToolbarGroup,
-  ToolbarItem,
 } from '@patternfly/react-core';
 import {
   Dropdown as DropdownDeprecated,
```
Comment (rhamilto) [inline frontend/public/components/monitoring/alerting.tsx:41]: @logonoff, these deprecated components need to be removed and replaced to fully satisfy https://issues.redhat.com/browse/OU-561.

**Pull request merged**